app/controllers/api/v1/welcome_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class WelcomeController < ApplicationController
+      before_action :config_locale, only: [:index]
```

Comment:
If we are always using the locale, why we are not applying this before action to every endpoint?

You can move this to application_controller and ejecute every request, and in the future if you do not want translations, you can do an exception of this callback.